components/service/glean/build.gradle

```diff
@@ -15,7 +15,7 @@ apply plugin: 'kotlin-android'
  * This uses a specific version of the schema identified by a git commit hash.
  */
 String GLEAN_PING_SCHEMA_GIT_HASH = "63155ec"
-String GLEAN_PING_SCHEMA_URL = "https://raw.githubusercontent.com/mozilla-services/mozilla-pipeline-schemas/$GLEAN_PING_SCHEMA_GIT_HASH/schemas/glean/baseline/baseline.1.schema.json"
```

Comment:
This change will need to be reverted and `GLEAN_PING_SCHEMA_GIT_HASH` will need to be updated once mozilla-services/mozilla-pipeline-schemas#296 is merged.